doc/src/advanced/custom_potential/external.md

```diff
@@ -0,0 +1,205 @@
+# Part I: Implementation of the logic in a separate package
+
+We start by creating a new package using `cargo`:
+
+```
+cargo new mie_potential
+cd mie_potential
+```
+
+Open `Cargo.toml` and add the lines
+```toml
+[dependencies]
+lumol-core = {git = "https://github.com/lumol-org/lumol"}
+```
+
+to add `Lumol` as a dependency to the package.
+To test if everything works, open the `lib.rs` file in `src` and add
+```rust
+extern crate lumol;
+```
+in the first line.
+
+Run `cargo build` or `cargo test` to check if errors occur.
+
+## Defining the struct
+
+For the first part of the tutorial, the complete code will be written into the `lib.rs` file.
+
+The energy function of the Mie potential reads
+
+$u(x) = \frac{n}{n-m} \left(\frac{n}{m}\right)^{m/(n-m)}\epsilon \left[ \left( \frac{\sigma}{x}\right)^n - \left( \frac{\sigma}{x}\right)^m \right]$
+
+where $x$ denotes the distance between two interaction sites $i, j$, with $x = x_{ij} = | \mathbf{r}_j - \mathbf{r}_i |$. The parameters of the potential are
+
+- $n, m$ the repulsive and attractive exponents, respectively,
+- $\epsilon$ the energetic paramater,
+- $\sigma$ the particle diameter or structural parameter.
+
+We start by defining the `struct` for our potential. Add the following lines to `lib.rs`:
+
+```rust
```

Comment:
Do you think it would be possible to directly include the relevant part of the file in `/tutorials` ? This would make it easier to prevent the code from bit rotting.

Reply:
I'm note sure what you mean here. How would you directly include that part of the file?

Reply:
Ok, I checked and this is not possible with markdown. So let's keep it this way for now.

I do this with sphinx: you have example files somewhere, and then you include them in your documentation. (see [here](https://github.com/chemfiles/chemfiles.py/commit/0576e9f5bb03a86a2e4554bb81445e8e27ff77db#diff-f1747d293233dbb0c4a0161f196a1917R29) for example). This allow to ensure that the documentation and the actual files stays in sync.

Reply:
Ah I see, yes i do that in sphinx too.